akka-docs/src/main/paradox/typed/reliable-delivery.md

```diff
@@ -28,13 +28,13 @@ To use reliable delivery, add the module to your project:
 Normal @ref:[message delivery reliability](../general/message-delivery-reliability.md) is at-most once delivery, which
 means that messages may be lost. That should be rare, but still possible.
 
-For interactions between some actors that is not acceptable and at-least once delivery or effectively once processing
+For interactions between some actors, that is not acceptable and at-least once delivery or effectively-once processing
```

Comment:
I think `efectively once` is fine. Take for example `at-least once` earlier in the sentence.

Reply:
Maybe we should ask a native speaker. To my understanding we should either write "at least once" or "at-least-once".

@seglo / @chbatey, what are your opinions on this?

Reply:
@renatocaval:

> Maybe we should ask a native speaker.

@helena

Reply:
I actually think it's `at-least-once` `at-most-once` and `exactly-once`, though in a given document this should be upheld across the board one way or another.

Reply:
I agree that we should use at-least-once at-most-once and exactly-once. Would be great if you can update in this PR @eloots

Reply:
Thanks for the feedback. Will update accordingly

Reply:
Updated all occurrences of xxx once to xxx-once ([d7b83b9](https://github.com/akka/akka/pull/28738/commits/d7b83b94b8efb467b0bf248a96be580f0b716db6))